wagtailio/utils/migrations/0002_auto_20220802_1706.py

```diff
@@ -0,0 +1,83 @@
+# Generated by Django 3.2.12 on 2022-08-02 17:06
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("utils", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
```

Comment:
praise: this is a cool use of the migrations functionality

alas, it is something that is out of scope of this feature